src/server/json_family.cc

```diff
@@ -157,6 +159,98 @@ OpResult<json> GetJson(const OpArgs& op_args, string_view key) {
   return decoder.get_result();
 }
 
+string ConvertToJsonPointer(string_view json_path) {
```

Comment:
can you please add a comment with an example of transformation, so that a reader will have a clear picture what the code does.